### PR TITLE
A couple extra bits for UI Reshuffle and Saving.

### PR DESCRIPTION
**Pull request by mstratford**
Hopefully this fixes a couple of things regarding smaller screens, as well as some other things.

Requires https://github.com/UniversityRadioYork/MyRadio/pull/944.

### CODE REVIEW

**Comment by mstratford** [comment]
This is super ready now ❤️

File: src/showplanner/ImporterModal.tsx [hunk 1]
```diff
@@ -11,6 +11,22 @@ interface ImporterProps {
...
+  useEffect(() => {
+    window.addEventListener(
+      "message",
+      (event) => {
+        if (!event.origin.includes("ury.org.uk")) {
+          console.log(event.origin);
+          return;
+        }
+        if (event.data === "reload_showplan") {
+          props.close();
```
Comment (markspolakovs) [inline src/showplanner/ImporterModal.tsx:24]: Shouldn't you, you know, actually reload the showplan?

ED: you do this in showplanner/index. Maybe worth a comment, just to clarify?

**Review by markspolakovs — COMMENTED** [review]
Code looks good, will give it a s:b:in now

**Review by markspolakovs — APPROVED** [review]
LGTM - noticed some bugs while testing, but none introduced in this PR :)

File: src/showplanner/ImporterModal.tsx (hunk 1 above)
Comment (mstratford) [reply src/showplanner/ImporterModal.tsx:24]: Done

**Pull request merged**